atlasdb-commons/build.gradle

```diff
@@ -12,7 +12,7 @@ dependencies {
     compile project(":commons-executors")
     compile 'javax.ws.rs:javax.ws.rs-api:2.0.1'
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations'
-    compile group: 'io.dropwizard.metrics', name: 'metrics-core'
+    compile group: 'com.codahale.metrics', name: 'metrics-core'
```

Comment:
done. also added excludes in a few other places that i noticed. basically any project that depends on dropwizard-core i made exclude the older codahale one where necessary.  I'm not sure if theres an automated way to enforce this, but you can use the gradle dependenyInsight to verify that codahale is no longer appearing in the deps for those projects.